sdks/java/extensions/google-cloud-platform-core/pom.xml

```diff
@@ -35,22 +35,6 @@
 
   <build>
     <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-jar-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>build-unshaded-jar</id>
```

Comment:
We actually have some use cases where we need an unshaded jar. What was the problem here?

Reply:
I am actually not removing the functionality, if you look at the current file un master this definition of the jar plugin is repeated twice (and shows a warning), I am just removing such repetition where it happened.

Reply:
Oh, that's strange. I know it was first added to a profile which turned out wrong. But it sure does just look exactly duplicated.